ext/microprofile/mp-rest-client/src/main/java/org/glassfish/jersey/microprofile/restclient/MethodModel.java

```diff
@@ -210,13 +211,17 @@ private Form handleForm(Object[] args) {
         return form;
     }
 
-    private Object synchronousCall(Invocation.Builder builder, Object entity, Method method) {
+    private Object synchronousCall(Invocation.Builder builder,
+                                   Object entity,
+                                   Method method,
+                                   MultivaluedMap<String, Object> customHeaders) {
         Response response;
 
         if (entity != null
                 && !httpMethod.equals(GET.class.getSimpleName())
                 && !httpMethod.equals(DELETE.class.getSimpleName())) {
-            response = builder.method(httpMethod, Entity.entity(entity, consumes[0]));
+            String contentType = (String) customHeaders.get(HttpHeaders.CONTENT_TYPE).get(0);
```

Comment:
This should be:
`String contentType = (String) customHeaders.getFirst(HttpHeaders.CONTENT_TYPE);`